python/__init__.py

```diff
@@ -187,6 +187,7 @@ class Panda:
   FLAG_TOYOTA_ALT_BRAKE = (1 << 8)
   FLAG_TOYOTA_STOCK_LONGITUDINAL = (2 << 8)
   FLAG_TOYOTA_LTA = (4 << 8)
+  FLAG_TOYOTA_SECOC_CAR = (8 << 8)
```

Comment:
```suggestion
  FLAG_TOYOTA_SECOC = (8 << 8)
```

Reply:
not resolved?

Reply:
my bad, missed these on my search-and-replace sweep